src/modules/partition/core/PartitionLayout.cpp

```diff
@@ -164,60 +164,118 @@ PartitionLayout::execute( Device* dev,
                           const PartitionRole& role )
 {
     QList< Partition* > partList;
+    QMap< const PartitionLayout::PartitionEntry *, qint64 > partSizeMap;
```

Comment:
This could use a comment explaining that it is the .. um .. calculated minimum size for each partition, or 0 if the partition is defined only as percentages?

Reply:
Indeed, added a small comment to explain it a little bit